cypress/e2e/models/migration/task-manager/task-manager.ts

```diff
@@ -59,4 +70,11 @@ export class TaskManager {
                 cy.get(tasksStatusColumn).contains(status, { timeout: 30 * SEC });
             });
     }
+
+    public static applyAndValidateFilter(filterType: taskFilter, filterValue: string) {
```

Comment:
This method is called `applyAndValidateFilter` but it just applies the filter, the validations are performed in the test itself. So this method can be renamet to just `applyFilter`